src/ralph/data_importer/widgets.py

```diff
@@ -135,11 +136,16 @@ class ImportedForeignKeyWidget(widgets.ForeignKeyWidget):
     def clean(self, value):
         result = None
         if value:
-            content_type, imported_obj = get_imported_obj(self.model, value)
-            if imported_obj:
-                result = self.model.objects.filter(
-                    pk=int(imported_obj.object_pk)
-                ).first()
+            if settings.IMPORT_DATA_FROM_OLD_RALPH:
+                content_type, imported_obj = get_imported_obj(
+                    self.model, value
+                )
+                if imported_obj:
+                    result = self.model.objects.filter(
+                        pk=int(imported_obj.object_pk)
```

Comment:
don't know if mapping it always to int is a good idea - what if we'll use for example uid somewhere?